src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const title = document.querySelectorAll('th');
+
+for (const titleName of title) {
+  titleName.addEventListener('click', (e) => {
+    const item = e.target;
+    const tbody = document.querySelector('tbody');
+    const rows = [...tbody.querySelectorAll('tr')];
+
+    switch (item.innerText) {
+      case 'Age' || 'Salary':
+        const i = [...title].indexOf(item);
+
+        rows.sort((a, b) =>
+          Number(a.children[i].innerText.replace(/[, $]/g, ''))
+          - Number(b.children[i].innerText.replace(/[, $]/g, '')));
+
+        for (const row of rows) {
+          tbody.append(row);
+        }
```

Comment:
DRY. There is a duplicate code in lines 28-30

do this operation outside `switch case`